week2/homework/src/index.js

```diff
@@ -1,3 +1,71 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+
+const { readFile, writeFile } = require('fs');
+const { promisify } = require('util');
+
+const readTodo = promisify(readFile);
+const writeTodo = promisify(writeFile);
+
+const path = 'todo_list.json';
+
+async function app() {
+
+    const [, , cmd, ...args] = process.argv;
```

Comment:
I like that you extracted the command separately from the other arguments and gave it a proper name 👍